src/com/fediroryshchuk/homework7/brokesinglton/singletonpattern/SeniorGovernmentOfficials.java

```diff
@@ -0,0 +1,7 @@
+package com.fediroryshchuk.homework7.brokesinglton.singletonpattern;
+
+public class SeniorGovernmentOfficials {
```

Comment:
The class should be final with a private constructor.

Reply:
If I make this class final, I can’t extend it, but all my classes actually extend it.